### PR TITLE
Make junit categories work better when running tests across multiple projects

### PR DESCRIPTION
**Pull request by benmccann**
Make junit categories work better when running tests across multiple projects. Even for a single project, this is probably closer to the desired behavior when referencing a class that does not exist

@jroper this new feature seems to be working pretty well for us! Wanted to share this one small improvement

### CODE REVIEW

**Comment by jsuereth** [comment]
@benmccann This appears to make tests run twice.... can you look at the failure?

**Comment by benmccann** [comment]
Ok. Just as a heads up, I probably won't have a chance to revisit for while.

**Comment by jsuereth** [comment]
no problems.  As you've seen it takes me a while to circle through all the projects and get back here too :)

File: src/main/java/com/novocode/junit/JUnitRunner.java [hunk 1]
```diff
@@ -172,6 +177,17 @@ private boolean shouldRun(Fingerprint fingerprint, Class<?> clazz, RunSettings s
...
   }
 
+  /**
+   * Ignores any ClassNotFoundException
+   */
+  private static Set<Class<?>> loadClassesSilently(ClassLoader classLoader, Set<String> classNames) {
+    try {
+      return loadClasses(classLoader, classNames);
+    } catch (ClassNotFoundException e){
+      return new HashSet<Class<?>>();
```
Comment (jsuereth) [inline src/main/java/com/novocode/junit/JUnitRunner.java]: +1

**Comment by eed3si9n** [comment]
Closing this due to inactivity.

**Pull request closed without merging**